clients/src/main/java/org/apache/kafka/common/requests/CreateTopicsResponse.java

```diff
@@ -70,9 +70,12 @@ public int throttleTimeMs() {
     @Override
     public Map<Errors, Integer> errorCounts() {
         HashMap<Errors, Integer> counts = new HashMap<>();
-        data.topics().forEach(result ->
-            updateErrorCounts(counts, Errors.forCode(result.errorCode()))
-        );
+        for (CreateTopicsResponseData.CreatableTopicResult result : data.topics()) {
+            Errors error = Errors.forCode(result.errorCode());
+            if (error != Errors.NONE) {
```

Comment:
Not sure why we need this change. I think the convention is to include `NONE` in error counts.

Reply:
I guess there are some inconsistency between different RPCs as I spotted cases excluding NONE. I would initiate a separate JIRA for the cleaning and revert the change here.

Reply:
https://issues.apache.org/jira/browse/KAFKA-10607